kahuna/public/js/search/index.js

```diff
@@ -109,6 +115,7 @@ search.config(['$stateProvider', '$urlMatcherFactoryProvider',
                 // keep the shortcut keys here to stop overriding
                 return new Map([
                     ['metadataPanel', 'm'],
+                    ['collectionsPanel', 'c'],
```

Comment:
Can we put bucket (and basket and tag and route) behind us and not bring them back please? :pray: